### DISCUSSION TITLE
umi打包报错

### DISCUSSION THREAD

**Discussion by jxp1147** [umijs/umi#7782] [Q&A]
umi.js from UglifyJs
Unexpected token punc «{», expected punc «(» [umi.js:58773,10]

**Comment by Jianjie96** [comment]
碰到相同问题，minimizer改成terserjs好了
https://v2.umijs.org/config/#minimizer

**Comment by Jianjie96** [comment]
> umi.js from UglifyJs Unexpected token punc «{», expected punc «(» [umi.js:58773,10]

碰到相同问题，minimizer改成terserjs好了
https://v2.umijs.org/config/#minimizer
